src/main/webapp/assets/modules/salary-report/sr-survey-form.js

```diff
@@ -0,0 +1,60 @@
+techlooper
+  .directive("srSurveyForm", function ($http, $translate) {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/salary-report/sr-survey-form.tem.html",
+      link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.